### PR TITLE
Signing in setup

### PR DESCRIPTION
**Pull request by Hafizalimi7**

### CODE REVIEW

**Review by jmcg015 — APPROVED** [review]

File: client/src/pages/SignIn/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+
+// export const SignIn = () => {
+
+//   const [name, setNAme] = useState("GFG");
+
+//   const handleChange = () => ();
+
+
+//   return(
```
Comment (sophiegilder) [inline client/src/pages/SignIn/SignIn.js:17]: Can all these comments be removed?

File: client/src/pages/SignIn/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+	if (email === '' || password === '') {
+	setError(true);
+	} else {
+	setSubmitted(true);
+	setError(false);
+	}
+};
+
+// Showing success message
+const successMessage = () => {
```
Comment (sophiegilder) [inline client/src/pages/SignIn/SignIn.js:66]: Can this be changed to show the homepage or something once the log in is successful?

**Review by sophiegilder — CHANGES_REQUESTED** [review]

File: client/src/pages/SignIn/SignIn.js (hunk 1 above)
Comment (Hafizalimi7) [reply client/src/pages/SignIn/SignIn.js:17]: yep :)

File: client/src/pages/SignIn/SignIn.js (hunk 2 above)
Comment (Hafizalimi7) [reply client/src/pages/SignIn/SignIn.js:66]: yes, we could add a link when we connect to the database

File: client/src/pages/SignIn/SignIn.js (hunk 1 above)
Comment (Hafizalimi7) [reply client/src/pages/SignIn/SignIn.js]: done !

**Review by sophiegilder — APPROVED** [review]

**Pull request merged**